admin/css/views/events.css

```
@@ -1291,15 +1291,87 @@ html.theme-dark .event-filter-source-menu {
+.event-weather-detail-icon {
+    font-size: 13px;
+    line-height: 1;
+}
+
+.event-weather-detail-title {
+    letter-spacing: 0.01em;
+}
+
+.event-weather-detail-body {
+    white-space: pre-wrap;
+    word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)